checks.d/openstack.py

```diff
@@ -333,6 +357,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
 
         self._ssl_verify = init_config.get("ssl_verify", True)
+        self._use_proxy = init_config.get("use_agent_proxy", True)
```

Comment:
I think both are legit so it's a matter of choosing the default that is going to fit the most likely scenario. I'd leave it as it is but open to change it if we receive strong feedback by users to do this differently.